### PR TITLE
perf: sumac release support

### PR DESCRIPTION
**Pull request by luisfelipec95**
## Description

This PR aims to support the new Sumac release and removes tests and constraints for Python 3.8 due to compatibility issues.

## Testing instructions

1, Create a Sumac environment using Tutor or TVM.
2. Install eox-hooks using this branch you can follow the official [tutor documentation](https://docs.tutor.overhang.io/configuration.html#installing-extra-xblocks-and-requirements).
3. Follow the [test cases ](https://docs.google.com/document/d/1f7LkXyxUbBcAdkDsWhtEzNfVNFFmV65enpKrEUUnirw/edit?tab=t.0)file.

### CODE REVIEW

**Comment by luisfelipec95** [comment]
The test cases have already been tested in the remote sumac environment

File: README.rst [hunk 1]
```diff
@@ -112,8 +112,10 @@ Compatibility Notes
 +---------------------+----------------+
```
Comment (magajh) [inline README.rst:112]: Since we are dropping support for python 3.8, we need to add the constraint in the compatibility table for Quince < 8.0.0

**Comment by magajh** [comment]
@luisfelipec95 as the doc suggests, you need to review and merge the dependabots PRs as well, so we can keep our workflows updated https://github.com/eduNEXT/eox-hooks/pull/68

**Review by magajh — CHANGES_REQUESTED** [review]
@luisfelipec95 
we need to update the setup.py file
https://github.com/eduNEXT/eox-hooks/blob/master/setup.py#L80

Also, it's very important to always check the constraints in the project. In this case I believe, since we are removing the support for Python 3.8, that this constraint is no longer needed: https://github.com/eduNEXT/eox-hooks/blob/master/requirements/constraints.txt#L20-L21
So we need to remove it and then run `make upgrade` for the requirements

**Review by magajh — APPROVED** [review]
LGTM

**Review by jignaciopm — APPROVED** [review]
@luisfelipec95 @magajh I have tried the [test cases](https://docs.google.com/document/d/1f7LkXyxUbBcAdkDsWhtEzNfVNFFmV65enpKrEUUnirw/edit?tab=t.0) in the instance https://tenant-a.sumac.releases.edunext.link/ and all the results were successful.

**Pull request merged**